draft-ietf-bidding-and-auction-services.md

```diff
@@ -471,7 +472,8 @@ tuple.
    larger than request.
 1. Encrypt `request` using the `public key` and its `key id` as in
    {{request-encryption}} to get the `encrypted message` and `hpke context`.
-1. Let the `request context` be the tuple (`included_groups`, `hpke context`)
+1. Let the `request context` be the tuple
+   (`included_groups`, `hpke context`, `ig pagg coordinators`).

```

Comment:
Would it make more sense to expand `included_groups` to be a list of (name, pagg coordinator) pairs? That would avoid the extra redundancy.

Reply:
Do you mean let included_groups[owner]'s value be a list of (ig, coordinator) pairs? Sure we can do that, with an extra step to combine `interest group list` and `coordinator`. If that sounds good, I can make the change.

Reply:
the field is only used by this spec in "parse response", so it needs to be available in `request context`. But if we want to add it to be part of included_groups (as what's being discussed in the other comment), then we don't need this extra field.